File: PSTodos.Model/Entities/Usuario.cs
using System;

namespace PSTodos.Model.Entities
{
    public class Usuario : EntityBase
    {
        public string Login { get; set; }
        public string Nome { get; set; }
        public string Email { get; set; }
        public string Senha { get; set; }
        public bool Ativo { get; set; }
        public DateTime DtInclusao { get; set; }
    }
}

File: PSTodos.Angular.Frontend/src/app/models/usuario.ts
export class Usuario {
    public id: number;
    public login: string;
    public nome: string;
    public email: string;
    public senha: string;
    public ativo: boolean;
    public dtInclusao: Date;

    constructor() {
        this.id = null;
        this.login = "";
        this.nome = "";
        this.email = "";
        this.senha = "";
        this.ativo = false;
        this.dtInclusao = new Date();
    }
}